### ISSUE TITLE
False positive on const missing comment

### ISSUE THREAD

**Issue by MarcelXia** [mgechev/revive#691]
**Describe the bug**
False positive : exported const TTLkey should have comment (or a comment on this block) or be unexported  

**To Reproduce**
```
const (
	TTLKey       = "ttl"    // Tome to live key
)
```
When the comment is above it, it won't complain, but when it is after it (same line) it complains.

Using revive 1.2.1
Command:
revive -formatter friendly -exclude ./vendor/... ./...

**Comment by chavacava** [comment]
Hi @MarcelXia, thanks for filling the issue.

**Comment by ifooth** [comment]
same issue +1

**Comment by chavacava** [comment]
Hi, I've created the PR #700 

GO parser makes the difference between _comment_ and _documentation_
```go
// this is "documentation"
const AConst = 1 // this is a "comment"
```
`revive` checks the content of _documentation_. For example in the following code it will complain if the documentation text does not start with the name of the constant
```go
const (
	// Time to live key
	TTLKey       = "ttl"    
)
```

The PR, applies the same checks to _comments_, i.e. on 

```go
const (
	TTLKey       = "ttl"    // Time to live key
)
```
it will complain because the comment does not of the form `TTLKey ...`
Another thing, if a declaration has a _documentation_ and a _comment_ it will check only the documentation
```go
const (
	// TTLKey is the time to live key
	TTLKey       = "ttl"    // always in lowercase
)
```

What do you think?

**Comment by chavacava** [comment]
> I'm not sure, have to check myself, but I think the Go parser "knows" the comments or documentation belongs to a command. For my (being a silly human) the fact that it is clear by the comment above or after the line is enough.

Sure, the parser attaches _documentation_ and _comments_ to statements.
My question is about the pertinence of enforcing on _comments_ the same style rules that are enforced on _documentation_ (case 3 below)

There are the following cases:
Case 1: no doc, no comment
```go
const (
	TTLKey       = "ttl"  
)
```
`revive` will complain about the absence of doc

Case 2: doc, no comment
```go
const (
	// TTLKey is the time to live key
	TTLKey       = "ttl"
)
```
`revive` will complain iff the **doc** style is not respected (it must start with `TTLKey ...`)

Case 3: no doc, comment
```go
const (
	TTLKey       = "ttl" // TTLKey is the time to live key
)
```
`revive` will complain iff the **comment** style is not respected (it must start with `TTLKey ...`)

Case 4: doc and comment
```go
const (
	// TTLKey is the time to live key
	TTLKey       = "ttl"    // always in lowercase
)
```
`revive` will complain iff the **doc** style is not respected (it must start with `TTLKey ...`). 
Comment style is not checked.

That behavior is OK for you?
